utils.py

```diff
@@ -73,13 +73,14 @@ def parse_config(config):
     logging.info("load_options config: %s", config["load_options"])
 
     load_options = config.get("load_options")
+    batch_size = load_options.get("batch_size") if load_options.get("batch_size") else 1
     concurrency = load_options.get("concurrency")
     duration = load_options.get("duration")
 
     plugin_type = config.get("plugin")
     if plugin_type == "openai_plugin":
         plugin = openai_plugin.OpenAIPlugin(
-            config.get("plugin_options")
+            config.get("plugin_options"), batch_size
```

Comment:
Again, this should be handled in a way the the plugin does not need to know the batch size in advance. However, if you must... just set `config["plugin_options"] = batch_size` to avoid changing the interface.